packages/jest-cli/src/__tests__/generate_empty_coverage.test.js

```diff
@@ -4,14 +4,13 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
- *
- * @emails oncall+jsinfra
  */
 'use strict';
 
 const os = require('os');
 
 const generateEmptyCoverage = require('../generate_empty_coverage');
+const {makeGlobalConfig, makeProjectConfig} = require('../../../../test_utils');
```

Comment:
maybe we could setup `moduleNameMapper` and use it like `require('test_utils')`?